.github/workflows/ready_for_review.yml

```diff
@@ -1,9 +1,9 @@
 name: Pull Request Ready for Review
 on:
   pull_request:
-     types: [opened, reopened, ready_for_review, converted_to_draft, review_requested, review_request_removed, labeled, unlabeled]
+     types: [opened, reopened, synchronize, ready_for_review, converted_to_draft, review_requested, review_request_removed, labeled, unlabeled]
   workflow_run:
-    workflows: ["Code Checks", "Check CODEOWNERS Entries", "Code Health Report", "Audit Service Tags"]
+    workflows: ["Code Checks", "Check CODEOWNERS Entries", "Audit Service Tags"]
     types: [completed]
```

Comment:
Synchronize was added because we are pulling all workflow run statues for every commit to the pull request.

Reply:
"Code Health Report" workflow has been inactive for 9 months.